sentences/de/_common.yaml

```diff
@@ -52,8 +52,8 @@ expansion_rules:
   auf: "(auf | hoch | rauf | nach oben)"
   zu: "(zu | [her]runter | nach unten)"
   schliessen: "(schließ | schließe | zumachen)"
-  machen: "(mach[e|en])"
-  setzen: "(setz[e|en] | stell[e|en] | einstellen | änder[e|en]| veränder[e|n])"
+  machen: "( mach | mach[e|en])"
```

Comment:
```suggestion
  machen: "(mach([e|en]))"
```
this does the same

Reply:
i need to revise my statement, because `mach[e|en]` does already match `mach` , `mache` und `machen`